docker/gdpr/entry.sh

```diff
@@ -1,5 +1,19 @@
 #!/bin/sh
 
+#set environment
+export PGHOST=$POSTGRES_HOST
+export PGPORT=$POSTGRES_PORT
+export PGDATABASE=$POSTGRES_DB
+export PGUSER=$POSTGRES_USER
+export PGPASSWORD=$POSTGRES_PASSWORD
+
+#print environment
+echo "Started with psql config"
```

Comment:
please do never log DB username or password. Please test that this works when started using docker and then remove these log output lines.